test/application_system_test_case.rb

```diff
@@ -4,6 +4,20 @@
 require 'capybara/accessible'
 require 'selenium-webdriver'
 
+# rubocop:disable Layout/CommentIndentation
+Capybara::Accessible::Auditor.severe_rules = [
+  'AX_HTML_01',  # The web page must have the content's human language must be
+                 # indicated in the markup
+  'AX_HTML_02',  # An element's ID must be unique in the DOM
+  'AX_TEXT_01',  # Controls and media elements should have labels
+  'AX_TEXT_02',  # Images should have an alt attribute, unless they have an ARIA
+                 # role of 'presentation'
+  'AX_FOCUS_03', # Avoid positive integer values for tabindex
+  'AX_COLOR_01', # Text elements should have a reasonable contrast ratio
+  'AX_TABLE_01', # Tables must have appropriate headers
+]
+# rubocop:enable Layout/CommentIndentation
+
```

Comment:
There are [several other checks](https://github.com/GoogleChrome/accessibility-developer-tools/wiki/Audit-Rules) — particularly relating to `aria-` attributes (which I don't even use). For many of them though, the description just reads 'stub', so I'm unclear whether they're actually implemented in the JS library.